modules/markup/html_internal_test.go

```diff
@@ -270,19 +277,12 @@ func TestRender_IssueIndexPattern_Document(t *testing.T) {
 		"repo":   "someRepo",
 		"style":  IssueNameStyleNumeric,
 	}
-
-	testRenderIssueIndexPattern(t, "#1", "#1", &RenderContext{
-		Ctx:   git.DefaultContext,
-		Metas: metas,
-	})
-	testRenderIssueIndexPattern(t, "#1312", "#1312", &RenderContext{
+	actual, err := RenderIssueTitle(&RenderContext{
 		Ctx:   git.DefaultContext,
 		Metas: metas,
-	})
-	testRenderIssueIndexPattern(t, "!1", "!1", &RenderContext{
```

Comment:
Why removed this test?

Reply:
It is wrong.

Reply:
It should use `RenderIssueTitle` to test, and `RenderIssueTitle` doesn't use any trick to parse the issue patterns.

Reply:
Take old tests back and renamed to TestRender_IssueIndexPattern_NoShortPattern in 863170a91a08733c1e101307ba211486357a8f1e